src/options/set.rs

```diff
@@ -492,10 +492,13 @@ fn set_widths(opt: &mut cli::Opt) {
             });
             (cli::Width::Fixed(width), true)
         }
-        None => (
-            cli::Width::Fixed(opt.computed.available_terminal_width),
-            true,
-        ),
+        None => {
+            match git_config.as_ref().unwrap().get::<usize>("delta.width")
```

Comment:
I believe this `unwrap()` is causing lots of tests to fail.

Reply:
Oh, sorry I hadn't run `make test`, and yes as you said, I find a lot of tests failed now. I'm totally sorry.

Reply:
No problem! When there's that many failures it's usually something easy to fix.